static/app/components/replays/table/replayTableColumns.tsx

```diff
@@ -47,24 +49,34 @@ import type {
 
 type ListRecord = ReplayListRecord | ReplayListRecordWithTx;
 
-interface RenderProps {
+interface HeaderProps {
+  columnIndex: number;
+  listItemCheckboxState: ReturnType<typeof useListItemCheckboxContext>;
+}
+
+interface CellProps {
   columnIndex: number;
   replay: ListRecord;
   rowIndex: number;
   showDropdownFilters: boolean;
 }
 
 export interface ReplayTableColumn {
-  Component: (props: RenderProps) => ReactNode;
-  name: string;
+  Component: (props: CellProps) => ReactNode;
+  Header: string | ((props: HeaderProps) => ReactNode);
   sortKey: undefined | ReplayRecordNestedFieldName;
-  tooltip?: ReactNode;
+  width?: string;
 }
 
 export const ReplayActivityColumn: ReplayTableColumn = {
-  name: t('Activity'),
-  tooltip: t(
-    'Activity represents how much user activity happened in a replay. It is determined by the number of errors encountered, duration, and UI events.'
```

Comment:
I replaced the separate `name` and `tooltip` fields with `Header` that returns a `ReactNode`, to support a checkbox being in the header cell.